setup_schema.json

```diff
@@ -47,6 +49,16 @@
             "examples": [
                 false, true
             ]
+        },
+        "ssl_ignore": {
+            "$id": "#/properties/ssl_ignore",
+            "type": "boolean",
+            "title": "The ssl_ignore schema",
+            "description": "Set true if you want to skip SSL verification",
+            "default": false,
+            "examples": [
```

Comment:
I would skip examples for simple boolean properties. It doesn't add any help to the user.